src/Array/reduce.js

```diff
@@ -0,0 +1,23 @@
+import { run } from "jest-cli";
+
+/**
+ * Returns ther resulting from the repeated application of the callback across a
+ * sliding window of the array.
+ * @param {callback} arg The function to be applied
+ * @param {initialValue} arg The starting value
+ *
+ * If initialValue is not provided, reduce() will execute the callback function
+ * starting at index 1, skipping the first index. If initialValue is provided,
+ * it will start at index 0.
+ *
+ */
+Array.prototype.myReduce = function (callback, initialValue) {
+  if (this.length == 0) {
+    return initialValue;
+  }
+  let runningValue = initialValue ? initialValue + this[0] : this[0];
```

Comment:
Should this be 
`initialValue ? callback (initialValue, this[0]) : this[0]`

The documentation says the following

>A value to use as the first argument to the first call of the callback. If no initialValue is supplied, the first element in the array will be used as the initial accumulator value and skipped as currentValue. Calling reduce() on an empty array without an initialValue will throw a TypeError